.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 12.13.0
```

Comment:
@tyankatsu0105 Could you shortly explain what this is doing? :slightly_smiling_face:

Reply:
( FWIW I think it should probably be deleted before we merge )

Reply:
Oh... This is my mistake.

Reply:
done https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50238/commits/f80a5430a990ae19b282179b7671326dd37e352b

Thanks for informing me!!